apps/test/unit/templates/StudentsBeyondHocTest.js

```diff
@@ -5,6 +5,19 @@ import StudentsBeyondHoc from '@cdo/apps/templates/StudentsBeyondHoc';
 import Responsive from '@cdo/apps/responsive';
 
 describe('StudentsBeyondHoc', () => {
+  it('renders a VerticalImageResourceCardRow component', () => {
+    const responsive = new Responsive();
+    const wrapper = shallow(
+      <StudentsBeyondHoc
+        completedTutorialType="other"
+        MCShareLink=""
+        responsive={responsive}
+        isRtl={false}
+      />
+    );
+    expect(wrapper.find('VerticalImageResourceCardRow').exists()).to.be.true;
```

Comment:
Can we start adding strong constraints to the React component tests?  It's fine if it's just validating the placeholder value for now — it'll be a good test to make sure they're updated along with any string changes.